src/main/java/edu/wpi/first/tableviewer/dialog/PreferencesDialog.java

```diff
@@ -113,8 +113,12 @@ private void startButtonPressed(java.awt.event.ActionEvent evt) {//GEN-FIRST:eve
                 String host = hostField.getText();
                 if (host.isEmpty()) {
                     return;
+                } else if (host.matches("\\d+") && Integer.valueOf(host) < 100000) {
+                    // 100000 is to filter for integer ip addresses
```

Comment:
How so? Not sure what 100,000 gets you. The current max for a team number (with 10.TE.AM.2) is 25599, if we're going to limit it we should choose that, not an arbitrarily-chosen value like 100,000.